### DISCUSSION TITLE
I keep getting this error

### DISCUSSION THREAD

**Discussion by eeel452** [victornpb/undiscord#502] [4. Bugs and Issues]
i downloaded it and then i go to delete all my msgs in personal dm and it says "Error searching messages, API responded with status 401!
	{"message":"401: Unauthorized","code":0}
CoreException	{} " Someone please help

**Comment by lawlingoctopus** [comment]
im also getting 
Error searching messages, API responded with status 401!
	{"message":"401: Unauthorized","code":0}

i dont know how to fix it. need some help please!

**Comment by wastu01** [comment]
I have same problem and fix by reinstall update version
https://github.com/victornpb/undiscord/issues/653

**Reply by lawlingoctopus** [reply]
cool thanks for the reply 
update the script ? or the tampermonkey extesion on google?
i did remove and reinstall it but it was from the same site and unsure if they were up to date 

i have manually delete since my posting but for future "cleaning house" i will go take a look and see what version i have vs whats "up to date"

**Reply by wastu01** [reply]
https://github.com/victornpb/undiscord/issues/653
